tool/tctl/common/resource_command.go

```diff
@@ -1138,14 +1138,14 @@ func (rc *ResourceCommand) createServerInfo(ctx context.Context, client auth.Cli
 
 	exists := (err == nil)
 	if !rc.force && exists {
-		return trace.AlreadyExists("server info %q already exists", name)
+		return trace.AlreadyExists("Server info %q already exists", name)
```

Comment:
nit: Don't capitalize Go errors, these are typically meant to be chained. Note the difference between an `error` and an actual complete message, like the fmt.Printf we have below - the latter I'd encourage you to capitalize.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```suggestion
		return trace.AlreadyExists("server info %q already exists", name)
```